kombu/utils/scheduling.py

```diff
@@ -27,13 +33,16 @@ class FairCycle:
         predicate (type): Exception predicate.
     """
 
-    def __init__(self, fun, resources, predicate=Exception):
+    def __init__(self, fun: Callable[[ResourceType, CallbackType],
+                                     FunReturnType],
```

Comment:
I do not know how to specify that there are `**kwargs` at the end that can be anything.